command/commands/user.py

```diff
@@ -74,6 +74,8 @@ def init_subparsers(self):
         parser_edit.add_argument("--major", type=str, action='store')
         parser_edit.add_argument("--bio", type=str, action='store')
         parser_edit.add_argument("--member", type=str, action='store')
+        parser_edit.add_argument("--perm", type=lambda x: Permissions[x],
```

Comment:
I find `--perm` a little bit non-obvious - I feel like "perm" isn't really a common shortening of "permissions."

Reply:
I agree. I would just put `--permissions` instead

Reply:
since you are not looking at your permissions that often anyway, it doesn't hurt to write the whole thng.